src/ip.cpp

```diff
@@ -130,6 +125,7 @@ IP::IP(const uint8_t* buffer, uint32_t total_sz) {
             options_.push_back(option(opt_type));
         }
     }
+
```

Comment:
nit: Unnecessary blank line.

(The author doesn't use much whitespace (or any?) within functions. I'm not a big fan of that style, but we should try to follow his style.)